apps/public-docsite-v9/package.json

```diff
@@ -37,6 +37,7 @@
     "@fluentui/theme-designer": "*",
     "@fluentui/react-swatch-picker-preview": "*",
     "@fluentui/react-motions-preview": "*",
+    "@fluentui/react-nav-preview": "*",
```

Comment:
how this landed here ? 

for adding any stage of v9 package to our official docs we have automated process which needs to be followed

- https://github.com/microsoft/fluentui/wiki/Component-Implementation-Guide#preview-release
- https://github.com/microsoft/fluentui/blob/master/tools/workspace-plugin/src/generators/prepare-initial-release/README.md